docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
+The root directory of your CumulusCI project. Because each project is linked to a single GitHub repository, CumulusCI knows which project you are working on by the current working directory of your shell. 
 
-Automation Configuration (``cumulusci.yml``)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. tip:: Avoid headaches by making sure you're in the correct repository for your project before running project-specific commands; otherwise, your project will produce an error. (This tends to be one of the biggest troubleshooting issues that developers run into with CumulusCI. Check for this first and you might save yourself an extra trip to this guide.).
 
-``cumulusci.yml`` is the definition of the project's automation. It includes information about the project itself as well as customizations that define how scratch orgs are built and provide other important functionality to be used as part of the project's lifecycle.
+In order to be used as a CumulusCI project, a directory must both be a Git repository and contain a ``cumulusci.yml`` configuration file. We'll show you how to get set up with a new or existing CumulusCI project in the [TODO link Get Started section].
 
-When you work with CumulusCI in a project, the contents of your ``cumulusci.yml`` are merged with the ``cumulusci.yml`` that comes with CumulusCI and defines out-of-the-box functionality. From time to time we'll refer to that as the standard library. Learn more about customizing CumulusCI automation in TODO: link Customization section.
+cumulusci.yml
+^^^^^^^^^^^^^
 
-Project Source Code
-^^^^^^^^^^^^^^^^^^^
+The ``cumulusci.yml`` file defines a project's automation in CumulusCI. It contains all the customizations and configurations that pertain to your project's lifecycle. It can encompass everything from customizing the shapes of scratch orgs to configuring tasks and flows.
 
-The main body of the project's code and metadata lives in the ``force-app`` directory for Salesforce DX-format projects and ``src`` for Metadata API-format projects. For managed package projects, only this metadata is part of the package. Most projects have the vast majority of their components stored here.
+Learn more about customizing CumulusCI automation in the [TODO: link Customization section].
 
-Orgs
-^^^^
+force-app (or src)
+^^^^^^^^^^^^^^^^^^
+
+The main body of the project's code and metadata lives in the default package directory, commonly shown as the ``force-app`` folder for Salesforce DX-format projects and the ``src`` folder for Metadata API-format projects. **force-app** defines what will be included when you release a managed package from your CumulusCI project (or when you release an unlocked package, or if you're not releasing the package at all but running the ``deploy`` task to get the metadata into an org in unmanaged form).
```

Comment:
```suggestion
The main body of the project's code and metadata lives in the default package directory, commonly shown as the ``force-app`` folder for Salesforce DX-format projects and the ``src`` folder for Metadata API-format projects. **force-app** defines what's included when you release a managed package from your CumulusCI project. Or, when you release an unlocked package, or if you're not releasing the package at all but running the ``deploy`` task to get the metadata into an org in unmanaged form.
```

Reply:
I often use parentheticals as asides, meaning not vital but still letting the reader in the know. Again, I've written this entire page as if I'm giving a presentation aloud. So any emphasis, whether bold, italics, em-dashes or parentheses, gives a rhythm to the writing. I'm not saying this because the text is precious. But if we remove all of the parentheses, we're completely stripping away the "confidential tone" that I'm using to engage the reader.

Perhaps we can discuss with JV after the holidays and find a happy medium?

Reply:
I fear that I'm giving off "I don't like parentheticals" vibes when actually, I do like them! In this case, it was more about sentence length getting long, making for more work to parse. Breaking up the sentence with a harder stop feels like it gives me permission to set down that first chunk and then consider the next chunk. The parenthetical seems to ask for more cognitive holding of that first chunk. Anyway, this is getting really esoteric and nitpicky. Just note a potential readability issue that may or may not be a real issue. (And happy to get JV's take on it.)

Reply:
I turned the parenthetical into its own sentence. It looks a bit odd, but I still don't think it has enough weight to drop the parentheses. Additionally, it's something Parker mentioned specifically, so I'm hesitant to drop it completely because it _is_ helpful to the reader with a certain skill level.

Tagging @jvarese so he knows what we're talking about next week.

Reply:
@justinvholt - I like parenthetical, though Liane's right about us needing a more definite break to the flow. Sounds like making it its own sentence is a great solution. If the semi-colon comes back after that first clause, that might help too. (And I do admire your circumspection around the parenthetical -- a difficult thing to master.)

Reply:
@justinvholt - a PS: if you don't want the **forceapp** that starts the sentence to look weird, you could simply redundantly start the sentence with "The forceapp folder/directory defines . . . " I know we are always trying to write less words, not more, but this is one of those cases where need for clarity trumps that ambition.